src/libmongoc/src/mongoc/mongoc-thread-private.h

```diff
@@ -29,11 +29,20 @@
 #define mongoc_cond_t pthread_cond_t
 #define mongoc_cond_broadcast pthread_cond_broadcast
 #define mongoc_cond_init(_n) pthread_cond_init ((_n), NULL)
+
+#ifndef MONGOC_ENABLE_TESTING
 #define mongoc_cond_wait pthread_cond_wait
+#else
```

Comment:
This should only be defined if on Unix.

Reply:
ping

Reply:
Sorry I should have commented on this earlier.  I didn't change this because it is within the `#if defined(BSON_OS_UNIX)` block.  Should add a check for that in the inner if also?

Reply:
Ah nevermind. I had missed that, and in my other comment in `common-thread-private.h`. Resolving.